bcl2fastq/index_summary.py

```diff
@@ -297,5 +211,49 @@ def update_pie_chart(run_alias, known_json, unknown_json):
     )
```

Comment:
It is very hard to visually see that this return statement gives back two values. Reformat it with newlines/spacing so it is more obvious.